### DISCUSSION TITLE
Auth via Clerk

### DISCUSSION THREAD

**Discussion by ds2600** [Peppermint-Lab/peppermint#129] [Ideas]
Taking a look at the next branch, it looks like it's heavily centered around auth, or at least a lot of what I looked at was 😁
Would love for Clerk integration, just because it allows a little more flexibility and interoperability with potentially other systems.

**Comment by potts99** [comment]
Currently single sign on is going to use Keycloak to allow for custom providers, i might look at a different solution to this to allow for custom single sign on but this is the easiest currently

**Reply by potts99** [reply]
also clerk wouldnt be very selfhostable as it relies as a user having an account with an externally hosted company

**Reply by ds2600** [reply]
Understood. Just thinking as an option, as it has a very high level free tier that would allow some flexibility. Either way, I'm excited for the next version! Great job on everything, something with this ease of use was missing for sure.

**Comment by potts99** [comment]
SSO is now implemented via next-auth.js 👍
